diff --git a/dist/client/components/router-reducer/fetch-server-response.js b/dist/client/components/router-reducer/fetch-server-response.js
index 5c216ecfa94ab533a3d4f8e7ac89be852283a038..9e48df4a3889adcea6d0250421b200478cc060b6 100644
--- a/dist/client/components/router-reducer/fetch-server-response.js
+++ b/dist/client/components/router-reducer/fetch-server-response.js
@@ -478,6 +478,7 @@ async function createFetch(url, headers, fetchPriority, shouldImmediatelyDecode,
         headers[_approuterheaders.NEXT_REQUEST_ID_HEADER] = crypto.getRandomValues(new Uint32Array(1))[0].toString(16);
     }
     const fetchOptions = {
+        cache: 'no-store',
         // Backwards compat for older browsers. `same-origin` is the default in modern browsers.
         credentials: 'same-origin',
         headers,
diff --git a/dist/esm/client/components/router-reducer/fetch-server-response.js b/dist/esm/client/components/router-reducer/fetch-server-response.js
index bb2793b6687d9c5c3496197c03a77f1f7e3ab6ba..ac25ec12306f75fc71a261a11e4eef33cb2b1559 100644
--- a/dist/esm/client/components/router-reducer/fetch-server-response.js
+++ b/dist/esm/client/components/router-reducer/fetch-server-response.js
@@ -478,6 +478,7 @@ export async function createFetch(url, headers, fetchPriority, shouldImmediately
         headers[NEXT_REQUEST_ID_HEADER] = crypto.getRandomValues(new Uint32Array(1))[0].toString(16);
     }
     const fetchOptions = {
+        cache: 'no-store',
         // Backwards compat for older browsers. `same-origin` is the default in modern browsers.
         credentials: 'same-origin',
         headers,
